python/shotgun_fields/image_widget.py

```diff
@@ -312,18 +380,16 @@ def _update_btn_position(self):
 
     def _update_display(self):
         """
-        If no pixmap, display a link to upload if the widget is editable,
-        otherwise display a ``No Image`` string.
+        If no pixmap, display the button with options.
```

Comment:
is part of the docstring missing? This seems like a second line to me, with the summary line missing!